client/asset/btc/btc.go

```diff
@@ -611,11 +623,8 @@ type baseWallet struct {
 	lastPeerCount     uint32
 	peersChange       func(uint32)
 	minNetworkVersion uint64
-	fallbackFeeRate   uint64
-	feeRateLimit      uint64
+	config            *baseWalletConfig
```

Comment:
Could you please embed this so you don't have to add `.config.` everywhere. It'll actually save me a bunch of conflicts in #1686 if you do.